src/LiveApi.js

```diff
@@ -125,10 +130,7 @@ export default class LiveApi {
     }
 
     onClose() {
-        setTimeout(() => {
-            this.connect();
-            this.resubscribe();
-        }, 1000);
+        this.connect();
```

Comment:
I tested with no delay, working fine, and I think we need to have as little delay as possible (Still handshake is taking time)